### PR TITLE
Nextflow docs: BLAST example

### PR DESCRIPTION
**Pull request by hannahle**

### CODE REVIEW

File: docs/source/tutorials/nextflow.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+  - blast
+```
+
+You can create an environment called `blast-nf` using Micromamba like so:
+
+```console
+/root/bin/micromamba create -f /root/blast-nextflow-latch/blast-nf/conda.yml -y
+```
+
+You can verify the dependencies installation by listing our the environment:
```
Comment (jvfe) [inline docs/source/tutorials/nextflow.md:133]:
```suggestion
You can verify the dependencies installation by listing the environments:
```

File: docs/source/tutorials/nextflow.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+
+![Nextflow Dockerfile](../assets/nextflow-example/nextflow-dockerfile.png)
+
+* **Line 1**: is the [Latch base image](https://github.com/latchbio/latch-base), which is used to configure libraries required for consistent task behaviour.
+* **Line 3-4**: downloads and installs the updates for each outdated package and dependency on the machine that executes the workflow. `curl` and `unzip` are also installed.
+* **Line 7**: installs the Java runtime environment, which is required to run Nextflow.
+* **Line 8-10**: installs Nextflow and moves the binary to `/usr/bin`.
+* **Line 13-25**: is a series of commands to install Micromamba.
+* **Line 27**: copies the BLAST Nextflow pipeline code to the task execution environment. The `/root/blast-nf` is that path at which the NF code is stored in the machine that executes the task on Latch.
+* **Line 30**: uses Micromamba to install the dependencies as spefieid in `/root/blast-nf/conda.yml`.
```
Comment (jvfe) [inline docs/source/tutorials/nextflow.md:190]:
```suggestion
* **Line 30**: uses Micromamba to install the dependencies as specified in `/root/blast-nf/conda.yml`.
```

File: docs/source/tutorials/nextflow.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+latch register -r .
+```
+
+Then, open a remote debugging session:
+
+```console
+latch develop .
+```
+
+Enter an interative shell:
```
Comment (jvfe) [inline docs/source/tutorials/nextflow.md:207]:
```suggestion
Enter an interactive shell:
```

File: docs/source/tutorials/nextflow.md [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+```console
+apt-get install -y default-jre-headless
+```
+
+### Install Nextflow
+
+```console
+curl -s https://get.nextflow.io | bash && \
+    mv nextflow /usr/bin/ && \
+    chmod 777 /usr/bin/nextflow
```
Comment (maximsmol) [inline docs/source/tutorials/nextflow.md:88]:
```suggestion
    chmod u+x /usr/bin/nextflow 
```

File: docs/source/tutorials/nextflow.md [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+
+To expose workflow parameters to a user-friendly workflow GUI, you can use the `LatchMetadata` object. An important point to note is that all workflow arguments need to be added to the `parameters` key of LatchMetadata for them to display on the GUI. For an exhaustive list of how workflow argument and their Python types map to the front-end interface, visit [Customizing Your Interface](../basics/customizing_interface.md)
+
+```python
+"""The metadata included here will be injected into your interface."""
+metadata = LatchMetadata(
+    display_name="Example: Wrapping a Nextflow BLAST Pipeline in Latch SDK",
+    documentation="your-docs.dev",
+    author=LatchAuthor(
+        name="John von Neumann",
```
Comment (maximsmol) [inline docs/source/tutorials/nextflow.md:296]: ?

File: docs/source/tutorials/nextflow.md [hunk 6]
```diff
@@ -0,0 +1,398 @@
...
+        email="hungarianpapi4@gmail.com",
```
Comment (maximsmol) [inline docs/source/tutorials/nextflow.md:297]: ???

**Review by maximsmol — CHANGES_REQUESTED** [review]
a lot of comments from #216 apply here

---

`blast-nf-wf.png` has overlapping text somehow

---

also use unabbreviated command line parameters

**Pull request closed without merging**